lncrawl/binders/calibre.py

```diff
@@ -56,6 +56,12 @@ def epub_to_calibre(app, epub_file, out_fmt):
         file_name_without_ext,
         "--authors",
         app.crawler.novel_author,
+        '--synopsis', 
+        app.crawler.synopsis,
+        '--language', 
+        app.crawler.language,
+        '--novel-tags',
+        app.crawler.novel_tags,
```

Comment:
The calibre does not have these options. Check here for the available options: https://manual.calibre-ebook.com/generated/en/ebook-convert.html

Reply:
only the `--language` option is available

Reply:
The website linked describe `--comments` as "Set the e-book description" So I guess we should use that for synopsis. And the good options is `--tags` instead of `--novel-tags`